static/js/settings_emoji.js

```diff
@@ -257,7 +257,9 @@ export function set_up() {
                 return;
             }
 
-            if (is_default_emoji(emoji.name)) {
+            if (is_default_emoji(emoji.name.replace(/ /g, "_"))) {
+                // spaces are replaced with `_` to simulate how the emoji
+                // name will actually be stored in the backend
```

Comment:
Generally it's best for comments to appear before the code they're explaining, rather than after.

Reply:
I moved the logic into `is_default_emoji` and also resolved this while doing so.